package com.kodilla.exception.test;

import java.io.IOException;

/*public class ExceptionHandling<x, y>() extends Exception{

    *//*try() {
        (x >= 2 || x < 1 || y == 1.5)

    } catch(IOException e) {
        new ExceptionHandling();
    } finally() {*//*

    }
}*/
